pkg/network/status/status.go

```diff
@@ -50,8 +50,7 @@ const (
 	cleanupPeriod = 1 * time.Minute
 )
 
-// TODO(https://github.com/knative/serving/issues/6407):  Default timeouts may lead to hanging probes.
-var dialContext = (&net.Dialer{}).DialContext
+var dialContext = (&net.Dialer{Timeout: 5 * time.Second}).DialContext
```

Comment:
I think 5s is a bit high. I'd limit to 1s, probably

Reply:
Done